action.yml

```diff
@@ -11,7 +11,7 @@ inputs:
     required: true
   requirements-path:
     description: Path to requirements file, relative to `repo-directory`, to look for dependencies. Default will look for requirements.txt in the root of repo-directory
-    required: true
+    required: false
```

Comment:
What does this do? I would assume we "require" requirements.txt

Reply:
Poetry uses dependencies in `pyproject.toml`, it also supports .lock files like yarn and npm. So, we don't hav eto have a requirements.txt file.